js/src/layers/SVGOverlay.js

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) Jupyter Development Team.
+// Distributed under the terms of the Modified BSD License.
+
+const L = require('../leaflet.js');
+const rasterlayer = require('./ImageOverlay.js');
+
+const DEFAULT_LOCATION = [0.0, 0.0];
+
+export class LeafletSVGOverlayModel extends ImageOverlayModel {
```

Comment:
`ImageOverlayModel` is not defined at this point, I suppose you want to extends `rasterlayer.ImageOverlayModel` instead? Same comment for `ImageOverlayView` later.